README.md

```diff
@@ -48,8 +49,9 @@ Then, transform incoming params in your controller using `Decanter#decant`:
 # app/controllers/trips_controller.rb
 
   def create
-    trip_params = TripDecanter.decant(params[:trip])
-    @trip = Trip.new(trip_params)
+    trip_params = params.require(:trip) # or params[:trip] if you are not using Strong Parameters
+    transformed_trip_params = TripDecanter.decant(trip_params)
```

Comment:
`decanted_trip_params` or `parsed_trip_params`? Transformed is a little general (although I realize it's the language used at the top of the readme)